meteor/client/ui/Prompter/controller/joycon-device.ts

```diff
@@ -219,30 +225,36 @@ export class JoyConController extends ControllerAbstract {
 
 	private getActiveInputsOfJoycons(joycons: JoyconWithData[]): number {
 		let lastSeenSpeed = 0
+		let lastSeenSpeedTimestamp = 0
 
 		for (const joycon of joycons) {
-			// handle buttons at the same time as evaluating stick input
-			this.handleButtons(joycon)
-
-			// hadle speed input
-			if (joycon.mode === 'L' || joycon.mode === 'R') {
-				// L or R mode
-				if (Math.abs(joycon.axes[0]) > this.deadBand) {
-					if (joycon.mode === 'L') {
-						lastSeenSpeed = joycon.axes[0] * -1 // in this mode, L is "negative"
-					} else if (joycon.mode === 'R') {
-						lastSeenSpeed = joycon.axes[0] * 1.4 // in this mode, R is "positive"
+			// sort/filter by gamepad timestamp to use the most up-to-date input, in order to prevent the "stuck" dead joycon when going from pairs to singles
+			if (joycon.timestamp > lastSeenSpeedTimestamp) {
```

Comment:
So I had a little bit of a think when doing the dishes today, and I realized that we need to have a persistent (I mean across-frames-persistent, not stored in localStorage) `lastSeenSpeedTimestamp` variable, here's why:

Suppose you have two joycons, one active, the other one stuck in a +50 position. 
1. The Prompter is scrolling, because of the stuck one.
2. You move the active one away from deadband, starts scrolling at the speed of the active one, since that's the one with the latest timestamp
3. You move the active one back into the deadband, which then means that the stuck one has the latest timestamp again, and everything starts scrolling, because the current `lastSeenSpeedTimestamp` is scoped to this, single-frame function, `getActiveInputsOfJoycons`.

The thing is, with a persistent `lastSeenSpeedTimestamp` this comparison needs to be `joycon.timestamp >= lastSeenSpeedTimestamp`, because we actually want to keep scrolling when someone goes full bore and the gamepad value doesn't update.